org.kavil.gestimator.json

```diff
@@ -0,0 +1,106 @@
+{
+    "app-id": "org.kavil.gestimator",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.30",
```

Comment:
Is there a particular reason 3.32 doesn't work? Whilst the base for 3.30 is still in security support it doesn't really get fixes from the GNOME project any more.

Reply:
App works fine under 3.32. Will re-base to 3.32.